cpp/include/cudf/io/json.hpp

```diff
@@ -512,13 +512,15 @@ class json_reader_options_builder {
  * @endcode
  *
  * @param options Settings for controlling reading behavior
+ * @param stream Cuda stream to use for device memory operations
```

Comment:
I think most parts of cudf are using this docstring. Can we align the docstrings in this PR?
```suggestion
 * @param stream CUDA stream used for device memory operations and kernel launches
```

Reply:
Docstring aligned